common/djangoapps/student/helpers.py

```diff
@@ -122,7 +124,10 @@ def check_verify_status_by_course(user, course_enrollment_pairs, all_course_mode
         # verification status.
         if verified_mode is not None and enrollment.mode in CourseMode.VERIFIED_MODES:
             deadline = verified_mode.expiration_datetime
+
             relevant_verification = SoftwareSecurePhotoVerification.verification_for_datetime(deadline, verifications)
+            if relevant_verification is not None:
+                most_recent_verifications.append(relevant_verification.expiration_datetime)
```

Comment:
It would be slightly more space efficient to store just the maximum value rather than a list.